src/lib.rs

```diff
@@ -246,6 +134,71 @@ pub fn init_tracing() -> Result<(), SetGlobalDefaultError> {
     tracing::subscriber::set_global_default(subscriber)
 }
 
+/* Blocks operation */
+/// Filters for the first message of a particular identifier by block number
+/// get the timestamp it was received from and add the collection duration to
+/// return the time for which message comparisons should be triggered
+pub async fn comparison_trigger<
```

Comment:
Does this belong in the SDK? Seems like a Radio-specific implementation, just like the `NETWORKS` constant. Also it assumes that the Radio always stores messages which might not be the case

Reply:
This is a helper function usable in both POI radio and Query radio. For any radio that have multiple networks and eventual aggregation/comparison/attestation, this function should be helpful in triggering those actions. 

Also note that functions being in the SDK library doesn't mean they must be called in all radios